sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md

```diff
@@ -2850,6 +2859,22 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -EnableVersionExpirationSetting
+
+The EnableVersionExpirationSetting will allow to opt-in the file version expration feature, including setting version history limits on on organization level, site level, library level, creating job to trim existing versions on site/library, etc.  
```

Comment:
Use the EnableVersionExpirationSetting parameter to opt your tenant into public preview of Improved Version History controls feature Microsoft 365 Public Roadmap under ID [145802] (https://www.microsoft.com/en-us/microsoft-365/roadmap?filters=&searchterms=145802).

When set to true and feature roll out to your tenant has completed, Admin Version history controls at Organization, Site and Library levels will be available. When set to false, the feature will be disabled for your tenant. You will be able to toggle the value within the duration of the preview.